modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -94,20 +87,16 @@ TEST(TestAgeGenderIE, InferBasicTensor)
 
     IE::Blob::Ptr ie_age, ie_gender;
     {
-        IE::CNNNetReader reader;
-        reader.ReadNetwork(topology_path);
-        reader.ReadWeights(weights_path);
-        auto net = reader.getNetwork();
+        auto plugin        = cv::gapi::ie::wrap::getPlugin(params);
+        auto net           = cv::gapi::ie::wrap::readNetwork(params);
+        auto this_network  = cv::gapi::ie::wrap::loadNetwork(plugin, net, params);
```

Comment:
our internal stuff is usually under `cv::gimpl` prefix. `cv::gapi` is the public API part. Recommend to rename to avoid the confusion

Reply:
renamed